config/locales/en/helpers.yml

```diff
@@ -82,6 +84,9 @@ en:
           disqualification: You were banned from driving for a certain period of time
           endorsement: You were found guilty of a motoring offence, which is now included on your driver record
           penalty_points: You were given number of penalty points for a driving offence
+        conviction_length:
+          weeks: weeks
```

Comment:
I imagine we don't need hints for these radios.
Then you can do as I did for `custodial_sentence`, empty strings to not show the hint.